Sources/PackageDescription4/Package.swift

```diff
@@ -10,8 +10,10 @@
 
 #if os(Linux)
 import Glibc
-#else
+#elseif os(iOS) || os(macOS) || os(tvOS) || os(watchOS)
```

Comment:
canImport Darwin?

Reply:
I really think that `canImport` for that is really bad.  What happens when I have the `Darwin` module on Windows?  Or the `Darwin` module on Linux?  Why should that be given precedence?  Why not give MSVCRT precedence and default rights?

Reply:
I think we should focus on realistic scenarios rather than theoretical ones. Who would ever attempt to get the Darwin module working for a Windows or Linux target triple and under what motivation? This is not a simple task, and there's also no reason I can think of why anyone would ever want that. Do you have a use case?